Documentation/ProjectFile/prj/time_loop/global_process_coupling/convergence_criteria/t_convergence_criterion.md

```diff
@@ -0,0 +1,2 @@
+Defines the convergence criteria of the global un-coupling loop of the staggered
```

Comment:
"un-coupling loop" is not a common phrase. Do you mean weakly coupled processes:
"Defines the convergence criterion for each weakly coupled process." --- would be this correct?

Reply:
Changed accordingly.